src/ui/3d/qgspointcloud3dsymbolwidget.ui

```diff
@@ -132,6 +132,44 @@
         </property>
        </widget>
       </item>
+      <item row="2" column="0">
+       <widget class="QgsCollapsibleGroupBox" name="mTriangulateGroupBox">
+        <property name="title">
+         <string>Triangulate Points</string>
+        </property>
+        <property name="checkable">
+         <bool>true</bool>
+        </property>
+        <layout class="QGridLayout" name="gridLayout_8">
+         <property name="leftMargin">
+          <number>0</number>
+         </property>
+         <property name="topMargin">
+          <number>0</number>
+         </property>
+         <property name="rightMargin">
+          <number>0</number>
+         </property>
+         <property name="bottomMargin">
+          <number>0</number>
+         </property>
+         <item row="0" column="0">
+          <widget class="QCheckBox" name="mFilterTriangleBySizeCheckBox">
+           <property name="text">
+            <string>Filter triangles by size</string>
```

Comment:
How about *"Maximum triangle size"* instead? So that it's obvious that the biggest triangles are filtered out.